ext/CMakeLists.txt

```diff
@@ -43,12 +43,21 @@ if (HAERO_BUILDS_EKAT)
     set(EKAT_ENABLE_FPE OFF CACHE BOOL "")
   endif()
   if (HAERO_ENABLE_GPU)
+    if (HAERO_AMD_GPU)
+      set(Kokkos_ENABLE_HIP ON CACHE BOOL "Enable HIP Kokkos backend")
+    else()
+      set(Kokkos_ENABLE_CUDA ON CACHE BOOL "Enable CUDA Kokkos backend")
+      set(Kokkos_ENABLE_CUDA_LAMBDA ON CACHE BOOL "Enable CUDA lambdas")
+      set(EKAT_NVCC_WRAPPER ${CMAKE_BINARY_DIR}/bin/nvcc_wrapper)
+      set(EKAT_NVCC_WRAPPER ${EKAT_NVCC_WRAPPER} PARENT_SCOPE)
+      file(COPY ${CMAKE_CURRENT_SOURCE_DIR}/ekat/extern/kokkos/bin/nvcc_wrapper
+           DESTINATION ${CMAKE_BINARY_DIR}/bin)
+    endif()
     set(Kokkos_ENABLE_SERIAL ON CACHE BOOL "Enable serial Kokkos backend")
-    set(Kokkos_ENABLE_CUDA ON CACHE BOOL "Enable CUDA Kokkos backend")
-    set(Kokkos_ENABLE_CUDA_LAMBDA ON CACHE BOOL "Enable CUDA lambdas")
   else()
     set(Kokkos_ENABLE_SERIAL ON CACHE BOOL "Enable serial Kokkos backend")
     set(Kokkos_ENABLE_CUDA OFF CACHE BOOL "Disable CUDA Kokkos backend")
```

Comment:
The comments for these options should probably say "Enable", not disable. They're off by default, but the help string is just a description of the option when set to `ON`, isn't it?